src/core/agent/services/multiSigService.test.ts

```diff
@@ -1406,6 +1407,98 @@ describe("Multisig sig service of agent", () => {
     expect(result.threshold).toBe(3);
   });
 
+  test("Should return member identifiers that have rotated ahead of multisig", async () => {
+    Agent.agent.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(true);
+    const multisigIdentifier = "newMultisigIdentifierAid";
+    const signifyName = "newUuidHere";
+    identifiersGetMock = jest.fn().mockResolvedValue(aidReturnedBySignify);
+    identifiersCreateMock = jest.fn().mockResolvedValue({
+      identifier: multisigIdentifier,
+      signifyName,
+    });
+    identifierStorage.getIdentifierMetadata = jest
+      .fn()
+      .mockResolvedValue(keriMetadataRecord);
+    mockResolveOobi = jest.fn().mockResolvedValue({
+      name: "oobi.AM3es3rJ201QzbzYuclUipYzgzysegLeQsjRqykNrmwC",
+      metadata: {
+        oobi: "testOobi",
+      },
+      done: true,
+      error: null,
+      response: {},
+      alias: "c5dd639c-d875-4f9f-97e5-ed5c5fdbbeb1",
+    });
+
+    identifiersMemberMock = jest.fn().mockResolvedValue({
+      signing: [
+        {
+          aid: "ENYqRaAQBWtpS7fgCGirVy-zJNRcWu2ZUsRNBjzvrfR_",
+          ends: {
+            agent: {
+              EGQnU0iNKuvURoeRenW7pZ5wA1Iyijo2EgscSYsK0hum: {
+                http: "http://dev.keria.cf-keripy.metadata.dev.cf-deployments.org:3902/",
+              },
+            },
+          },
+        },
+        {
+          aid: "EOpnB724NQqQa58Zqw-ZFEQplQ2hQXpbj6o2gKrzlix3",
+          ends: {
+            agent: {
+              "EAOfcPsG_mHtrzw1TyOxlCiQQlLZn-KTUu4lUy7zB_Na": {
+                http: "http://dev.keria.cf-keripy.metadata.dev.cf-deployments.org:3902/",
+              },
+            },
+          },
+        },
+      ],
+      rotation: [],
+    });
+
+    queryKeyStateMock.mockImplementation((id: string) => {
+      if (id === "ENYqRaAQBWtpS7fgCGirVy-zJNRcWu2ZUsRNBjzvrfR_") {
+        return {
+          name: "oobi.AM3es3rJ201QzbzYuclUipYzgzysegLeQsjRqykNrmwC",
+          metadata: {
+            oobi: "testOobi",
+          },
+          done: true,
+          error: null,
+          response: {
+            i: "id",
+            k: ["k1"],
+          },
+        };
+      }
+      return {
+        name: "oobi.AM3es3rJ201QzbzYuclUipYzgzysegLeQsjRqykNrmwC",
+        metadata: {
+          oobi: "testOobi",
+        },
+        done: false,
+        error: null,
+        response: {
+          i: "id2",
+        },
+      };
+    });
```

Comment:
These are still a bit funky. Can you please add real query key state responses? One of these has `k: ["k1"]`, another has nothing. And the responses have `oobi` everywhere.

The tests need to be as accurate as possible or they aren't properly testing it.

Reply:
@Sotatek-BaoHoanga It's still not correct overall. We have `EGvWn-Zv7DXa8-Te6nTBb2vWUOsDQHPdaKshNUMjJssB` with `k=DIH7-xjcUC-xPS9I32b0ftZAT6gHJvfHiBR4UwxtWuEO` and `EJwDuZ8YpU-1g6QVwioZG-PmyufLXaDHXvfFLWkqENeL` with `k=DI3bh31vfuGyV14LvtBxHHljnDnSqbKQ7DZ9iiB_51Oh`.

The test I asked for earlier was where 1 has rotated and the other hasn't.

This means the multi-sig identifier must have `k` of length 2, where one of the values of `k` equals either `DIH7-xjcUC-xPS9I32b0ftZAT6gHJvfHiBR4UwxtWuEO` or `DI3bh31vfuGyV14LvtBxHHljnDnSqbKQ7DZ9iiB_51Oh`, and the other is some other value.

`aidReturnedBySignify` only has `k=["k"]` - this is NOT a multi-sig. Can we make sure all tests are properly using a multi-sig?